portfolio/src/main/webapp/index.html

```diff
@@ -1,30 +1,45 @@
 <!DOCTYPE html>
-<html style="background-color:Tomato;">
+<html>
   <head>
     <meta charset="UTF-8">
     <title>  Taniece's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body id="background">
       <img src="/Untitled/IMG_4578.jpeg" alt= "Taniece" 
         style="float:left;width:265px;height:400px;"> </img>
-        
-      <button id="button"  type= " button" onclick="addRandomFact()"> Click Me for a random fun fact!</button>
+      <button id="button"  type= " button"
```

Comment:
"button" is not a descriptive name for an id

Reply:
How about facts-button?

Reply:
sure